lib/index.js

```diff
@@ -1,346 +1,61 @@
-const { makeExecutableSchema } = require('graphql-tools');
-const { graphql } = require('graphql');
-const fs = require('fs');
-const path = require('path');
+const fs = require('fs'),
```

Comment:
Ditto, we use to declare one `const` per line.
Also the `'use strict'` directive is missing